dataactcore/scripts/initialize.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import logging
+import os
+
+
+from dataactvalidator.app import createApp
+from dataactbroker.scripts.setupEmails import setupEmails
+from dataactcore.models.userModel import User
+from dataactcore.interfaces.function_bag import createUserWithPassword
+from dataactcore.scripts.setupAllDB import setupAllDB
+from dataactbroker.handlers.aws.session import SessionTable
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.config import CONFIG_BROKER, CONFIG_DB
+from dataactvalidator.scripts.loadTas import loadTas
+from dataactvalidator.filestreaming.sqlLoader import SQLLoader
+from dataactvalidator.filestreaming.schemaLoader import SchemaLoader
+from dataactvalidator.scripts.loadFile import loadDomainValues
+from dataactvalidator.scripts.load_sf133 import load_all_sf133
+
+logger = logging.getLogger(__name__)
+basePath = CONFIG_BROKER["path"]
+validator_config_path = os.path.join(basePath, "dataactvalidator", "config")
+
+
+def setup_db():
+    """Set up broker database and initialize data."""
+    logger.info('Setting up db')
+    setupAllDB()
+    setupEmails()
+    setup_session_table()
+
+
+def create_admin():
+    """Create initial admin user."""
+    logger.info('Creating admin user')
+    admin_email = CONFIG_BROKER['admin_email']
+    admin_pass = CONFIG_BROKER['admin_password']
+    with createApp().app_context():
+        sess = GlobalDB.db().session
+        user = sess.query(User).filter(User.email == admin_email).one_or_none()
+        if not user:
+            # once the rest of the setup scripts are updated to use
+            # GlobalDB instead of databaseSession, move the app_context
+            # creation up to initialize()
+            user = createUserWithPassword(
+                admin_email, admin_pass, Bcrypt(), permission=2)
```

Comment:
Oof, good catch. My local user is already set up, so I wasn't hitting this. Good call on running this w/ an empty database.